concent_api/utils/logging.py

```diff
@@ -0,0 +1,71 @@
+from logging                        import getLogger
+
+from golem_messages.message.base    import Message
+
+from utils.helpers                  import get_task_id_from_message
+
+
+logger = getLogger(__name__)
+
+
+def log_message_received(message: Message, client_public_key: str):
+    logger.info('A message has been received in `send/` -- MESSAGE_TYPE: {} -- TASK_ID: {} -- CLIENT PUBLIC KEY: {}'.format(
+        message.TYPE,
+        get_task_id_from_message(message) or '',
```

Comment:
But in this case we definitely want to skip 0 or empty values. Ints would be ok in this case, they would get formatted correctly.

Reply:
Not really. If `task_id` were an integer, then 0 would be a valid ID. We wouldn't want to arbitrarily skip it.

In this particular case we don't have to worry about 0 but only because we assume that it's never an int. And if you make an assumption like that, you should add an `assert`.

Reply:
:+1: